packit.yaml

```diff
@@ -8,8 +8,14 @@ downstream_package_name: cockpit-podman
 copy_upstream_release_description: true
 
 actions:
-  post-upstream-clone: make cockpit-podman.spec
-  create-archive: make dist
+  post-upstream-clone:
+    - make node-cache dist
+
+  fix-spec-file:
+    # our locally built tarballs are not on the upstream releases page, point to the local files
+    - sh -c 'sed -i "/^Source1:/ s/https:.*/$(ls *-node*.tar.xz)/" cockpit-*.spec'
```

Comment:
```
[jelle@carbon][~/projects/cockpit-podman]%sed -i "/^Source1:/ s/https:.*/$(ls *-node*.tar.xz)/" cockpit-*.spec
sed: -e expression #1, char 85: unterminated `s' command
```

But I'm a tad bit confused, this only applies to Source1, while both sources come from a https url? But why is this even a problem?

Mental note, we have the same sed in cockpit-files.

Reply:
Adjusted in the same way as https://github.com/cockpit-project/cockpit-files/pull/1250 , and consistent with cockpit PR.